stacker/providers/aws.py

```diff
@@ -0,0 +1,98 @@
+import logging
+
+import boto
+from boto import cloudformation
+
+from .base import BaseProvider
+
+logger = logging.getLogger(__name__)
+
+
+class Provider(BaseProvider):
```

Comment:
yea, again, letting the filename specify the context. i can go more explicit if you'd like!